client/src/components/Disclaimer.js

```diff
@@ -1,37 +1,48 @@
 import React from 'react';
 import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
 import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
-import Button from 'react-bootstrap/Button';
 import { connect } from 'react-redux';
 import actions from '../redux/actions/actions';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
 
 export const Disclaimer = props => {
+  const [open] = React.useState(true);
```

Comment:
Do we need this if it's never being changed? Or could we just set `open={true}` since it will always be open in the sense that we already determine if we display the dialog at all with the `{!props.answer ...` conditional

Reply:
I only did this cause it kept giving me an error when I didn't. I tried it and it works when open={true}. Pushed up the change.

Reply:
Great!